.github/workflows/windows-rubygems.yml

```
@@ -33,6 +33,9 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby.value }}
           bundler: none
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

If you want to confirm my suspicion, try this:

```suggestion
          toolchain: stable
          target: x86_64-pc-windows-gnu
```